### ISSUE TITLE
Prestigable Skills

### ISSUE THREAD

**Issue by Forensiic** [ObliqueNET/Stratos#199]
**- Requested by Pharaoh**

Skill prestiging - Once you reach 100 in a skill you can reset it and go back to 1 for some prestige points. _You can only prestige a skill again after all skills have been prestiged._ 
Prestige shop - This is where you can spend your prestige points on things such as: vouchers, crates, random pokes+shinies, etc.

![image](https://user-images.githubusercontent.com/47290518/57567377-14ea6780-738d-11e9-9e67-954de391c6fa.png)

**Comment by JkaemtheGOAT** [comment]
Best suggestion I've seen in a long time

**Comment by Rasgnarok** [comment]
I'd love to see a list of 'prestige rewards', knowing that most of the rewards are already in crates/store/shopkeepers.

**Comment by fatyg5** [comment]
Indeed, as I've discussed in suggestions, good idea, needs work on actual benefits. No reason to prestige a skill and lose tons of mining, or catching speed, to get a small reward like crates. Unsure on _what_ could be given, but it's your idea so find out

**Comment by Forensiic** [comment]
**Prestigable Skills:** 

Catching - 20 points
Woodcutting - 30 points
Fishing - 10 points
Training - 20 points
Mining - 30 points

**Prestige Shop:**

_Vouchers_
1iv Voucher - 2 points
5iv Vouchers - 10 points
10iv Vouchers - 20 points
Gender Voucher - 10 points
Nature Voucher - 30 points
NPC Voucher - 10 points

_Keys_
Iron Key - 1 point
Gold Key - 2 points
Diamond Key - 4 points

_Random Pokemon_
Random Shiny - 3 points
Random Legendary - 100 points

**Comment by JkaemtheGOAT** [comment]
I think that list of rewards is pretty good, only suggestion I would have is to increase the Random Legendary point cost so it costs a little more than what you would get from prestiging your skills. For example, make the random legendary cost 120 points, which means people that have had every skill maxed out for months can't get a free legend right off the bat.

**Comment by Forensiic** [comment]
Prestiging all of your skills in exchange for a random legend that could turn out to be a celebi, meloetta or articuno is a big enough trade off. 100 points is good enough.

**Comment by SugarCandyyy** [comment]
Imagine getting a Phione >.>

**Comment by Archenemis** [comment]
Let's just say Phione isn't a legendary. Please?

**Comment by JkaemtheGOAT** [comment]
Phione is the best legend though?

**Comment by Archenemis** [comment]
I can imagine someone getting one now and then breeding it just to fill up Wonder Trade with even more of them..
